LondonBritishLibrary/add/BLadd19341.xml

```diff
@@ -97,7 +97,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <desc>Neatly written, around <date>1810</date>.</desc>
                             </handNote>
                             <handNote xml:id="h2" script="Latin" corresp="#a1">
-                              <desc>Apparentely the handwritig of the traveller.</desc>
+                              <desc>Apparently the handwriting of the traveller.</desc>
                             </handNote>
                         </handDesc>
 
```

Comment:
to line 109 below: qhat is encoded as `q `I guess should be `note` (Extracts...)

Reply:
No, I think it is fine like this: 
![image](https://user-images.githubusercontent.com/19650382/170999939-98e3e9b1-610f-4363-a333-aae48e57958b.png)